code/game/turfs/open/openspace.dm

```diff
@@ -150,6 +150,7 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
 	name = "ice chasm"
 	baseturfs = /turf/open/openspace/icemoon
 	initial_gas_mix = ICEMOON_DEFAULT_ATMOS
+	planetary_atmos = TRUE
 	var/replacement_turf = /turf/open/floor/plating/asteroid/snow/icemoon
```

Comment:
Will this have any effect on player made holes?

Reply:
True, player made holes will indeed leak the atmosphere. Not too sure how to resolve this but also complete the mission of the PR.